Gentoo bug: #359033

Avoid linking against system's. Gnucash-2.2.9 didn't have gnc_html_new, gnc_html_copy and gnc_html_export symbol.

--- a/src/bin/Makefile.am	2011-04-19 00:04:59.000000000 +0200
+++ b/src/bin/Makefile.am	2011-04-19 00:03:27.000000000 +0200
@@ -67,6 +67,7 @@
   ${top_builddir}/src/gnome-utils/libgncmod-gnome-utils.la \
   ${top_builddir}/src/app-utils/libgncmod-app-utils.la \
   ${top_builddir}/src/gnc-module/libgnc-module.la \
+  ${top_builddir}/src/report/report-gnome/libgncmod-report-gnome.la \
   ${GUILE_LIBS} \
   ${GLIB_LIBS} \
   ${GNOME_LIBS} \
